dev_guide/create_from_url.adoc

```diff
@@ -0,0 +1,73 @@
+[[dev-guide-create-from-url]]
+= Create From URL
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+[[overview]]
+== Overview
+
+Create from URL allows the construction of a URL that drops the user into the end
+(or near end) of the creation flow using an image stream and image tag or
+template.
+
+Create from URL only works with image streams or templates from
+namespaces that have been explicitly whitelisted.  The whitelist contains
+the 'openshift' namespace by default.  To add namespaces to the whitelist, see
+xref:../install_config/web_console_customization.adoc#configuring-the-create-
+from-url-namespace-whitelist[Configuring the Create From URL Namespace Whitelist].
+
+[[using-an-image-stream-and-image-tag]]
+== Using an Image Stream and Image Tag
+
+=== Query String Parameters
+
+[options="header"]
+|===
+|Name|Description|Required|Schema|Default
+|imageStream|The value metadata.name as defined in the image stream to be used.|
+true|string|
+|imageTag|The value spec.tags.name as defined in the image stream to be used.|
+true|string|
+|namespace|The name of the namespace containing the image stream and image tag
```

Comment:
seems redundant to say that it defaults to openshift when you have a whole column dedicated to the Default values